scene/feature-layer-rendering-mode-scene/README.md

```diff
@@ -1,31 +1,35 @@
-# Feature Layer Rendering Mode (Scene)
+# Feature layer rendering mode (scene)
```

Comment:
I think it's appropriate to leave Scene capitalised here: though strictly speaking Java should be "ArcGISScene" and that would mean renaming the samples etc...will leave it up to you what to go for!
```suggestion
# Feature layer rendering mode (Scene)
```

Reply:
We casually call it 'Scene' in many places, so I think it's ok not to call it ArcGISScene. The class name `ArcGISScene` is really just because `Scene` is already reserved by JavaFX.